code/datums/wires/shield_generator.dm

```diff
@@ -2,7 +2,7 @@
 	holder_type = /obj/machinery/power/shield_generator/
 	wire_count = 5
 	descriptions = list(
-		new /datum/wire_description(SHIELDGEN_WIRE_POWER, "This wire seems to be carrying a heavy current."),//, SKILL_EXPERT), //TODO: Hook in Eris skills here
+		new /datum/wire_description(SHIELDGEN_WIRE_POWER, "This wire seems to be carrying a heavy current."),//, STAT_LEVEL_EXPERT), //TODO: Hook in Eris skills here
```

Comment:
There's a TODO here

Reply:
This is not even closely related to this PR i just massively replaced values